tests/helpers.py

```diff
@@ -61,6 +65,45 @@ def open(self, *args, **kwargs):
         return super(CTFdTestClient, self).open(*args, **kwargs)
 
 
+class CTFtime:
```

Comment:
I don't know why it bothers me but I think this is maybe better as  `class ctftime` ?

Reply:
The function is currently called ctftime https://github.com/CTFd/CTFd/blob/master/CTFd/utils/dates/__init__.py#L7
so I tried to avoid using the same name as to not confuse the two (although just uppercasing may not be a big enough change either)